Codeinwp/templates-patterns-collection
# Release

### Summary
Supporting changes in line with the ones from https://github.com/Codeinwp/demo-data-exporter/pull/113

### Test instructions

1. Check that the count is not being displayed if templates < 50
2. Check with the Legacy Plugin that the count is visible as before.
3. Check that the Pallet Switch for global colors is available and working inside Preview


References: Codeinwp/demo-data-exporter#112.
